test/e2e/framework/util.go

```diff
@@ -1377,7 +1378,9 @@ retriesLoop:
 		// NOTE the test may need access to the events to see what's going on, such as a change in status
 		actualWatchEvents := scenario(resourceWatch)
 		errs := sets.NewString()
-		ExpectEqual(len(expectedWatchEvents) <= len(actualWatchEvents), true, "Error: actual watch events amount (%d) must be greater than or equal to expected watch events amount (%d)", len(actualWatchEvents), len(expectedWatchEvents))
+		if len(expectedWatchEvents) > len(actualWatchEvents) {
+			framework.Failf("Error: actual watch events amount", len(actualWatchEvents), "must be greater than or equal to expected watch events", len(expectedWatchEvents))
```

Comment:
Inside the "framework" package itself, `Failf` can be invoked without the `framework.` prefix.